docs/versioning.rst

```diff
@@ -46,6 +46,21 @@ Support levels
      - No support.
 
 
+.. _versioning_integration_library_support:
+
+Library support
+===============
+
+
+``ddtrace`` tries to support as many active versions of a library as possible, but in practice this is difficult
+due to the sheer maintenance burden of testing. The ddtrace library's support policy for third-party libraries that
```

Comment:
```suggestion
``ddtrace`` tries to support as many active versions of a library as possible.
The ddtrace library's support policy for third-party libraries that
```


internal detail customers shouldn't have to worry about 🤔 

I am not convinced that my change is required, just sharing a thought.